src/reason_pprint_ast.ml

```diff
@@ -630,9 +630,6 @@ let rules = [
     (TokenPrecedence, (fun s -> (Right, s = "asr")));
   ];
   [
-    (TokenPrecedence, (fun s -> (Left, s.[0] == '#')));
```

Comment:
What exactly is that rules array used for? I added it at the right place (just higher precedence that INFIXOP4)